docs/pages/reference/_meta.js

```diff
@@ -41,4 +41,5 @@ export default {
   "unstorage-adapter": "@auth/unstorage-adapter",
   "upstash-redis-adapter": "@auth/upstash-redis-adapter",
   "xata-adapter": "@auth/xata-adapter",
+  "test-adapter": "@auth/test-adapter",
```

Comment:
In order to really test `signIn` method I made a dummy test adapter 
<img width="963" alt="image" src="https://github.com/nextauthjs/next-auth/assets/49499/9b3daab1-dba0-4765-b08e-46ef6b3855c9">

Reply:
This can be used for other frameworks like Svelte or SolidJS to easy the testing

Reply:
great attempt, although I think no need for a new package, you could move this file: `packages/core/test/memory-adapter.ts` under `util` package & use it in `next-auth`, that should be enough

Reply:
I have a problem `core/test` is not exposed from `@auth/core`. How I can import. Indeed it was already a memory adapter 😂 

https://github.com/nextauthjs/next-auth/blob/main/packages/core/test/memory-adapter.ts

Reply:
This works but we're breaking the boundaries between packages. For that I though it would be cleaner to have a dedicated package that everyone can use. 

Anyway if this is fine I'll move on
<img width="835" alt="image" src="https://github.com/nextauthjs/next-auth/assets/49499/283554d5-ecbb-4d6b-850d-651ff050faa0">